src/annotator/sidebar.tsx

```diff
@@ -399,6 +399,17 @@ export class Sidebar implements Destroyable {
     annotationCounts(document.body, this._sidebarRPC);
     sidebarTrigger(document.body, () => this.open());
 
+    // Schedule update layout to be notified after sidebar transition, only
```

Comment:
What is the reason for deferring the event until the `transitionend` event? This could be problematic if `transitionend` is not emitted for any reason or if a transition occurs for a reason other than the sidebar opening in future.